pages/c/_cluster/_product/projectsnamespaces.vue

```diff
@@ -48,7 +48,7 @@ export default {
 
   computed: {
     isNamespaceCreatable() {
-      return (this.schema?.resourceMethods || []).includes('PUT');
+      return (this.schema?.resourceMethods || []).includes('POST');

```

Comment:
My 0.2c  - I think that you want `collectionMethods` here instead of `resourceMethods` as well as changing from `PUT` to `POST`.

Reply:
I do believe you are correct, thanks for the eyeballs on this!

Reference: https://github.com/rancher/api-spec/blob/master/specification.md